src/invoices/InvoiceDetails/constants.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export const ACCORDION = {
```

Comment:
I think it's a common constant, since we have the same accordions on details and edit. What do you think of moving this to the upper level?

Reply:
agreed